dcs/country.py

```diff
@@ -161,8 +157,8 @@ def reserve_onboard_num(self, number: int) -> bool:
         self._tail_numbers.add(number)
         return is_in
 
-    def next_onboard_num(self) -> int:
-        free_set = {x for x in range(10, 999)} - self._tail_numbers
+    def next_onboard_num(self) -> str:
+        free_set = {str(x) for x in range(10, 999)} - self._tail_numbers
```

Comment:
we should here atleast use the same format, as we did for serialization: `"{:03}"`

Reply:
Format has been changed to what it was originally being serialized as